src/porepy/models/energy_balance.py

```diff
@@ -355,6 +355,7 @@ def enthalpy_dirichlet(boundary_grids):
                 subdomains=subdomains,
                 dirichlet_operator=enthalpy_dirichlet,
                 neumann_operator=self.enthalpy_flux,
+                robin_operator=self.enthalpy_flux,
```

Comment:
Are you sure this is relevant for advective fluxes? I don't think Upwind handles Robin.

Reply:
Not sure, no. I do not know nearly enough about this, so when I went through the code after the Robin-relevant changes, I just adjusted all ``_combine_boundary_ooperators`` calls to match the new signature. If it is not relevant for certain fluxes, maybe the robin_operator in ``_combine_boundary_ooperators`` can be optional to pass?